src/templates/EditorialNewsletter/schema.js

```diff
@@ -72,41 +72,46 @@ const createNewsletterSchema = ({
     editorModule: 'link'
   }
 
-  const paragraph = {
-    matchMdast: matchParagraph,
-    component: Paragraph,
-    editorModule: 'paragraph',
-    editorOptions: {
-      formatButtonText: 'Paragraph'
-    },
-    rules: [
-      ...globalInlines,
-      link,
-      {
-        matchMdast: matchType('strong'),
-        component: ({ attributes, children }) => (
-          <strong {...attributes}>{children}</strong>
-        ),
-        editorModule: 'mark',
-        editorOptions: {
-          type: 'STRONG',
-          mdastType: 'strong'
-        }
+  const createParagraphRule = customComponent => {
+    return {
+      matchMdast: matchParagraph,
+      component: customComponent || Paragraph,
+      editorModule: 'paragraph',
+      editorOptions: {
+        formatButtonText: 'Paragraph'
```

Comment:
```suggestion
        formatButtonText: 'Paragraph',
        type: customComponent ? 'LISTP' : undefined
```

We need to set this custom type. Otherwise Publikator will not be able to render two different components (maybe change the abstraction to receive the type name via arg as well)